packages/portfolio-contract/test/supports.ts

```diff
@@ -17,6 +17,38 @@ export {
   makeFakeTransferBridge,
 } from '@agoric/vats/tools/fake-bridge.js';
 
+const testChainInfo = {
+  ...fetchedChainInfo,
+  noble: {
+    ...fetchedChainInfo.noble,
+    // Enable PFM on noble for tests
+    features: ['pfm'],
+    pfmEnabled: true,
+    connections: {
+      ...fetchedChainInfo.noble.connections,
+      // Patch noble.connections to add axelar-dojo-1 for test
```

Comment:
Do we expect a connection from Noble to Axelar in production?
I don't see one in [Noble's peers](https://mapofzones.com/zones/noble-1/peers?columnKey=ibcVolume&period=24h) on mapofzones.